### PR TITLE
Xantholne Wintercoat Fluff

### PR DESCRIPTION
**Pull request by Xantholne**
Wintercoat fluff, allows ears to poke through the hood and instead of hair magically disappeairng it's now laying on the outside/chest like any normal person would have it. Also different from most hoods to where ti doesn't look suffocating. As well as a bit more floofier so it actually looks like a wintercoat unlike the others.
e0ed419d552cbdf8e9130774d7ca2456
4afcb2456842b300a932403aafaadb9c
7cfddca96937da28f650ebd5400a9617
016a82b85a0c36399ddd1da1ca607291
Oh and this part I forgot, for non Captain, HoS, BS, NT Rep, Warden roles.

### CODE REVIEW

**Comment by FreeStylaLT** [comment]
you need to patent this
the hood is groundbreaking

**Comment by Xantholne** [comment]
There's actually a way to make it so ears poke through, by default they're set to hide. I'm not terribly sure how people would like that though for their winter coats :P

**Comment by Necaladun** [comment]
LGTM

(Patent Pending Copywrite Necaladun do not infringe)

**Comment by KasparoVy** [comment]
Furgonomics(TM)

**Comment by taukausanake** [comment]
Isn't this just a more poofy version of that nerdy tracksuit thing you got before-ish?

**Comment by DesolateG** [comment]
Slav squats

**Comment by Xantholne** [comment]
Pretty much yeah, not really a tracksuit though.

**Comment by DesolateG** [comment]
Love it

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -772,6 +772,30 @@
...
+/obj/item/clothing/suit/hooded/hoodie/fluff/xantholne // Xantholne: Meex Zwichsnicrur
+	name = "stripped winter coat"
+	desc = "A velvety smooth black winter coat with white and red stripes on the side."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "xantholne_wintercoat"
+	hoodtype = /obj/item/clothing/head/hood/fluff/xantholne
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
+	cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 10, rad = 0)
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: Remove the above two, please.

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -772,6 +772,30 @@
...
+
+
+/obj/item/clothing/head/hood/fluff/xantholne // Xantholne: Meex Zwichsnicrur
+	name = "black winter hood"
+	desc = "A black hood attached to a stripped winter coat."
+	icon = 'icons/obj/custom_items.dmi'
+	icon_state = "xantholne_winterhood"
+	body_parts_covered = HEAD
+	cold_protection = HEAD
+	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
```
Comment (Fox-McCloud) [inline code/modules/customitems/item_defines.dm]: remove the above two, please.

**Comment by Ralta** [comment]
Where's the N7 logo? 👀

**Comment by tigercat2000** [comment]
It's so adorable.

**Pull request merged**